zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -696,6 +696,18 @@ protected void startRequestThrottler() {
 
     }
 
+    /**
+     * This is working when in standalone mode.
+     * <p>
+     * The leader, follower, observer, readOnly have fished by themselves. You can see the Class below:
```

Comment:
Fished?

Reply:
@eolivelli 
what's the matter with you? I don't understand, please give a detailed comment.

Reply:
@eolivelli 
I have modified the spelling error.
Done